examples/example_codegg.py

```diff
@@ -44,7 +44,6 @@
 # Print and plot results
 print(o)
 
-o.plot(filename='codegg_forecast.png')
-o.animation(filename='codegg_forecast.gif')
-o.plot_vertical_distribution(filename='codegg_vertical.png')
-
+o.plot(filename='codegg_forecast.png', fast=True)
+o.animation(filename='codegg_forecast.mp4', fast=True)
+o.plot_vertical_distribution()
```

Comment:
Was filename intentionally left out here?

Reply:
Yes, this plotting method is interactive with slider, and does not support saving to file.